pkg/machine/e2e/stop_test.go

```diff
@@ -46,5 +48,14 @@ var _ = Describe("podman machine stop", func() {
 		Expect(err).ToNot(HaveOccurred())
 		Expect(stopAgain).To(Exit((0)))
 		Expect(stopAgain.outputToString()).To(ContainSubstring(fmt.Sprintf("Machine \"%s\" stopped successfully", name)))
+
+		// Stopping a machine should update the last up time
+		inspect := new(inspectMachine)
+		inspectSession, err := mb.setName(name).setCmd(inspect.withFormat("{{.LastUp}}")).run()
+		Expect(err).ToNot(HaveOccurred())
+		Expect(inspectSession).To(Exit(0))
+		lastupTime, err := time.Parse(time.RFC3339, inspectSession.outputToString())
+		Expect(err).ToNot(HaveOccurred())
+		Expect(lastupTime.After(starttime)).To(Equal(true))
```

Comment:
Since you have to repush anyway, use 
`Expect(lastupTime).To(BeTemporally(">", starttime))` to generate proper error message if this fails.